admin/demos/demo_tutorial.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+. ./demo-base.sh
```

Comment:
Am I missing something? I can't find this file right now.

Reply:
Sorry, this has just been a draft so far. Cleaned it up and pushed the rest. Ready for review.